src/components/services-view/services-view.controller.ts

```diff
@@ -180,4 +181,35 @@ export class ServicesViewController implements angular.IController {
   private updateActiveCardStyles() {
     this.$timeout(this.resizeExpansion, 50);
   }
+
+  private setRecentlyViewed() {
```

Comment:
In case it's not clear, I mean an AngularJS service like `RecentlyViewedItemsService`